include/zenoh-pico/config.h

```diff
@@ -54,6 +54,12 @@
 #define Z_FEATURE_RX_CACHE 0
 #define Z_FEATURE_UNICAST_PEER 1
 #define Z_FEATURE_AUTO_RECONNECT 1
+
+/**
+ * Default session lease in milliseconds.
+ */
+#define Z_TRANSPORT_LEASE 10000
```

Comment:
This belongs with the other configurable parameters (lines 23-26), and I don't think the comment is needed as to be consistent with the other parameters.